### ISSUE TITLE
Can't debug anymore in my old project

### ISSUE THREAD

**Issue by Wissperwind** [flutter/flutter-intellij#6159]
Hi,

Please look at this issue:
https://github.com/flutter/flutter/issues/103397

I have a problem with my debugger. If I create a new project, I can use the debugger like normal.
But in my daily work project, the debugger just runs over all breakpoints. They turn from red breakpoints into grey stop signs. What can I do to find out the problem and solve it?

**Comment by stevemessick** [comment]
I read through the original report. That looks very unfortunate. It is possible something got corrupted in the project and the fastest way to fix it *might* be to create a new project and copy all the content from the old project to the new project.

Before doing that there are some things to try. First, have you done `pub get` recently? How about `pub upgrade`? If that doesn't help, can you make a copy of your old project to experiment with? I'd like to know what happens if you delete the `.idea` directory, then launch the IDE. Also, try deleting the `build` directory then running your app. Another one to try is `.dart_tool` but if you delete it be sure to do `pub get` again. Let us know.

@helin24 do you have any other suggestions?

**Comment by helin24** [comment]
This is tough to debug without being able to reproduce. I believe if the breakpoints are turning grey, that means our file mapping didn't work in the VM service; however, it looks like there are cases where the breakpoints remain red but still don't get triggered. The implementation of file mapping in web is different from mobile, but it sounds like this project isn't set up to do mobile so we can't check whether this is web-specific. 

Can you send your IntelliJ logs? Help > Show logs in finder, then send us idea.log. This will contain file names of your project, so let me know if you'd prefer me to send you an email for them.

**Comment by bhanuka96** [comment]
same issue. debug point doesn't stop anymore with flutter 3.0

**Comment by helin24** [comment]
@bhanuka96 are you also in flutter web? I'm not able to reproduce in my small project for flutter 3.0+. If you can send me IntelliJ logs (instructions above: https://github.com/flutter/flutter-intellij/issues/6159#issuecomment-1125539843) that could help.

**Comment by bhanuka96** [comment]
@helin24 No, it's mobile. I upgraded  the `gradle`, I think that's why I cannot debug. 

`distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip`

`classpath 'com.android.tools.build:gradle:7.2.0'`

**Comment by helin24** [comment]
I'm not sure gradle would have an impact on debug breakpoints. Logs would be the most helpful for trying to narrow down where the problem is coming from.

**Comment by Wissperwind** [comment]
@helin24
I did pup get. It does not change everything.
pup upgrade too.
I deleted .idea .dart_lang and the build folder. It is still not working.

**Comment by Wissperwind** [comment]
[mymonitor.zip](https://github.com/flutter/flutter-intellij/files/8714056/mymonitor.zip)
Try yourself. I deleted my code from the project. Try to set a breakpoint at the print in the main. In my case it just runs over it.

**Comment by ArmandF-Eclipsed** [comment]
I'm also experiencing the same issue. 

Have tried:
Switching to master -> didn't help
Reinstalling dart & Flutter plugins -> didn't help
Reinstalled the SDK -> also didn't work for me

**Comment by helin24** [comment]
@Wissperwind thanks for the sample project! I was able to reproduce on my windows machine.  I'll look into why this behaves differently from my own examples.

**Comment by SilentThunder89** [comment]
Created a new flutter project and had the same issue
I'm using macos, latest android studio, flutter 3, and latest plugin version.

**Comment by Dkrinker** [comment]
I am experiencing the same issue.  Same issue on mobile and web.  Version info below.

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.0.0, on macOS 12.5 21G5027d darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 13.4)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2021.2)
[✓] VS Code (version 1.67.2)
[✓] Connected device (3 available)
[✓] HTTP Host Availability

• No issues found!

**Comment by helin24** [comment]
@SilentThunder89 @Dkrinker I'm not able to reproduce this myself on mac with the versions you mentioned, so can you send me your IntelliJ logs? Help > Show logs in finder, then send us idea.log. This will contain file names of your project, so let me know if you'd prefer me to send you an email for them.

**Comment by Dkrinker** [comment]
> @SilentThunder89 @Dkrinker I'm not able to reproduce this myself on mac with the versions you mentioned, so can you send me your IntelliJ logs? Help > Show logs in finder, then send us idea.log. This will contain file names of your project, so let me know if you'd prefer me to send you an email for them.

Can you please send me your email?

**Comment by helin24** [comment]
@Dkrinker My email is helinx@google.com

Can you also let me know the Dart version you're using for your project? This can be found in Preferences> Languages & Frameworks > Dart.

**Comment by helin24** [comment]
For anyone on macos - I suspect the issue is coming from having a space somewhere in the path of your project. Just now I tried making a hello flutter project in `~/Documents/tests/with space/hello_flutter_space` and I got the same failure as @Dkrinker (thanks for the logs!). I'll work on a solution for this, but for now you can try moving your project to a directory without spaces if that's convenient.

For the windows issue, there's an error thrown by the VM service, and I'm not sure what's causing that. I have a project that reproduces the problem and I'll look into this further. If you're blocked on this, unfortunately I think the best suggestion I have now is to drop back to the previous version of the plugin.

**Comment by Dkrinker** [comment]
> For anyone on macos - I suspect the issue is coming from having a space somewhere in the path of your project. Just now I tried making a hello flutter project in `~/Documents/tests/with space/hello_flutter_space` and I got the same failure as @Dkrinker (thanks for the logs!). I'll work on a solution for this, but for now you can try moving your project to a directory without spaces if that's convenient.

That worked for me!  Thank you

**Comment by annagrin** [comment]
The difference between this project and hello_world is the base change in the index.html. Works on my mac but not on windows - investigating...

**Comment by helin24** [comment]
@mullerch I ran into this issue while using Linux and chrome: https://github.com/flutter/flutter-intellij/issues/6193

But it sounds like you have a different problem, as my issue occurred regardless of whether the project involved a symlink. Could you open a separate issue and describe further (or include screenshots) what you mean by "the path is resolved at execution and the IDE is not able to match the unresolved and the resolved paths"?
